zeroex/asset_data_decoder.go

```diff
@@ -55,10 +68,20 @@ type ERC1155AssetData struct {
 type ERC20BridgeAssetData struct {
 	TokenAddress  common.Address
 	BridgeAddress common.Address
-	Calldata      []byte
+	BridgeData    []byte
+}
+
+type StaticCallAssetData struct {
+	StaticCallTargetAddress common.Address
+	StaticCallData          []byte
+	ExpectedReturnHashData  [32]byte
 }
 
-// MultiAssetData represents an MultiAssetData
+type CheckGasPriceStaticCallData struct {
+	MaxGasPrice *big.Int
+}
+
+// MultiAssetData represents a MultiAssetData
```

Comment:
Non-blocking: This file has a lot of the unproductive comments that I mentioned. We should delete this at some point.